### PR TITLE
check for pagination when loading group members

### PR DESCRIPTION
**Pull request by robotmiller**
We need to call `__get_and_get_all` instead of just `__get` to load group members. `__get_and_get_all` returns the JSON response as a list, rather than returning a Response object.

I also noticed when we read the `Link` response header, our tests give it a leading space (I assume I modeled it after our actual API responses) but we can strip out this whitespace.

There was also a failing bundle test since the empty class attributes get removed now.

### CODE REVIEW

File: tests/test_bundle.py [hunk 1]
```diff
@@ -1689,8 +1689,8 @@ def test_guru_markdown_blocks(self, g):
     bundle.zip()
 
     self.assertEqual(read_html("/tmp/test_guru_markdown_blocks/cards/1.html"), """<div class="ghq-card-content__markdown" data-ghq-card-content-markdown-content="%3Cdiv%20style%3D%22background-color%3A%23F89E91%3Bcolor%3A%234A1717%3Bpadding%3A1px%3Btext-align%3Aleft%3Bfont-size%3A16px%3Bmargin-bottom%3A16px%22%3E%0A%3Cp%20style%3D%22margin%3A%2016px%22%3Etest%20content%20%3Cstrong%3Eabcd%201234.%3C%2Fstrong%3E%3C%2Fp%3E%0A%3C%2Fdiv%3E" data-ghq-card-content-type="MARKDOWN">
-<div class="" style="background-color:#F89E91;color:#4A1717;padding:1px;text-align:left;font-size:16px;margin-bottom:16px">
-<p class="" style="margin: 16px">
+<div style="background-color:#F89E91;color:#4A1717;padding:1px;text-align:left;font-size:16px;margin-bottom:16px">
+<p style="margin: 16px">
```
[the remark this answers is not in the document]
Comment (robotmiller) [reply tests/test_bundle.py:1693]: I think that matches what I'm getting, the difference is yours has empty `class=""` attributes. I'm not sure why those attributes weren't getting removed before.

[the remark this answers is not in the document]
Comment (robotmiller) [reply tests/test_bundle.py:1693]: It might have something to do with how BeautifulSoup reads the class attribute. We have this code:

```
    old_class_list = el.attrs.get("class") or []
    new_class_list = list(filter(lambda c: c.startswith("ghq-"), old_class_list))

    if new_class_list:
      el.attrs["class"] = new_class_list
    elif old_class_list:
      # we only remove the 'class' attribute if it was there in the first place.
      del el.attrs["class"]
```

The initial HTML in the test has `class=""` so I can see how it's possible that'd get left alone (it wouldn't satisfy the `elif old_class_list` condition) but when I run this test and print out `old_class_list`, it has the value `[""]` which I'm guessing is truthy.
